src/room.ts

```diff
@@ -90,6 +98,15 @@ export class Room extends EventEmitter implements Sayable {
   //   return this.load(contactGetter)
   // }
 
+  private async readyAllMembers(memberList: RoomRawMember[]): Promise<void> {
+    let contact: Contact
+    for (let member of memberList) {
+      contact = Contact.load(member.UserName)
```

Comment:
change this line to:
`let contact = Contact.load(member.UserName)`
because it's only used in this closure.